ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -815,7 +817,7 @@ private class PostgresLedgerDao(
       .map(mapContractDetails)
 
   override def lookupActiveContract(contractId: AbsoluteContractId): Future[Option[Contract]] =
-    dbDispatcher.executeSql { implicit conn =>
+    dbDispatcher.executeSql(s"lookup active contract [$contractId]") { implicit conn =>
```

Comment:
Same doubt as before: how is an `AbsoluteContractId` printed? If it's a tagged type, would it be better to unwrap it for readability?

Reply:
`AbsoluteContractId` is a case class that wraps a `LedgerString` (which is a more fancy `java.lang.String`), so I'll log that wrapped string specifically. Thanks.

Reply:
Addressed in f0a779ed1.